jetty-home/src/main/resources/etc/jetty.conf

```
@@ -8,4 +8,4 @@
 # Each line in this file becomes an argument to start.jar
 # in addition to those found in the start.ini file
 # =======================================================
-jetty-started.xml
+--module=state,pid
```

just a niggle as in my mind the order is the other way (yeah I know they will be sorted):
```suggestion
--module=pid,state
```